scripts/example_end-to-end_script.sh

```diff
@@ -0,0 +1,126 @@
+done
+
+# Note --start=$start --end=$end would not work with ncremap
+#ncremap -P eam -j 1 --xtr_mth=mss_val --vrt_fl=${e2c_path}/grids/vrt_remap_plev19.nc  -O ${rgr_dir} -v ${raw_var_list} -I ${rgr_dir_vert} ${flags}
+
+# CMORIZE Atmosphere monthly plev variables
+e3sm_to_cmip -i ${rgr_dir} -o $result_dir  -v ${cmip_var_list} -t ${tables_path} -u ${metadata_path}
+
+#
+# atm high freq daily h1+
+input_path=${model_data}/v2.eam.h1_input
+flags='-7 --dfl_lvl=1 --no_cll_msr --clm_md=hfs'
+raw_var_list="TREFHTMN,TREFHTMX,PRECT,TREFHT,FLUT,QREFHT"
+cmip_var_list="tasmin, tasmax, tas, huss, rlut, pr"
+rgr_dir=${result_dir}/rgr_day
+native_dir=${result_dir}/native_day
+ncclimo -P eam -j 1 --map=${map_file} --start=$start --end=$end --ypf=$ypf --split -c $caseid -o ${native_dir} -O ${rgr_dir} -v ${raw_var_list} -i ${input_path} ${flags}
+
+# CMORIZE Atmosphere daily variables
+e3sm_to_cmip --freq day -i ${rgr_dir} -o $result_dir  -v ${cmip_var_list} -t ${tables_path} -u ${metadata_path}
+#
+# atm high freq 3hrly h1+
+input_path=${model_data}/v2.eam.h4_input
+flags='-7 --dfl_lvl=1 --no_cll_msr --clm_md=hfs'
+raw_var_list="PRECT"
+rgr_dir=${result_dir}/rgr_3hr
+native_dir=${result_dir}/native_3hr
+cmip_var_list="pr"
+ncclimo -P eam -j 1 --map=${map_file} --start=$start --end=$end --ypf=$ypf --split -c $caseid -o ${native_dir} -O ${rgr_dir} -v ${raw_var_list} -i ${input_path} ${flags}
+
+# CMORIZE Atmosphere 3hrly variables
+e3sm_to_cmip --freq 3hr -i ${rgr_dir} -o $result_dir  -v ${cmip_var_list} -t --tables-path ${e2c_path}/cmor/cmip6-cmor-tables/Tables -u ${metadata_path}
+
+## land monthly h0
+input_path=${model_data}/v2.elm_input/
+flags='-7 --dfl_lvl=1 --no_cll_msr'
+raw_var_list="LAISHA,LAISUN,QINTR,QOVER,QRUNOFF,QSOIL,QVEGE,QVEGT,SOILICE,SOILLIQ,SOILWATER_10CM,TSA,TSOI,H2OSNO"
+cmip_var_list="mrsos, mrso, mrfso, mrros, mrro, prveg, evspsblveg, evspsblsoi, tran, tsl, lai"
+rgr_dir=${result_dir}/rgr_lnd
+native_dir=${result_dir}/native_lnd
+# Note either include the extra variable landfrac or specify the file that has landfrac for subgrid scale mode to work.
+ncclimo -P elm -j 1 --var_xtr=landfrac --map=${map_file} --start=$start --end=$end --ypf=$ypf --split -c $caseid -o ${native_dir} -O ${rgr_dir} -v ${raw_var_list} -i ${input_path} ${flags}
+# Alternative ncclimo invocation
+#ncclimo -P elm -j 1 --map=${map_file} --start=$start --end=$end --ypf=$ypf --split -c $caseid -o ${native_dir} -O ${rgr_dir} -v ${raw_var_list} -i ${input_path} ${flags} --sgs_frc=${input_path}/v2.LR.historical_0101.elm.h0.1850-01.nc/landfrac
+
+#raw_var_list_elm_bgc="TOTLITC,CWDC,TOTPRODC,SOIL1C,SOIL2C,SOIL3C,^SOIL4C$,COL_FIRE_CLOSS,WOOD_HARVESTC,TOTVEGC,NBP,GPP,AR,HR"
+
+# CMORIZE Land Monthly variables 
+e3sm_to_cmip -i ${rgr_dir} -o $result_dir  -v ${cmip_var_list} -t --tables-path ${e2c_path}/cmor/cmip6-cmor-tables/Tables -u ${metadata_path}
+
+# CMORIZE Sea-ice Monthly variables 
+# Note the input folder for mpas sea ice files requires:
```

Comment:
@xylar I started to understand the e2c better. would you help review if the following is accurate? Could you let me know the Constants that are required from mpaso_in.

Reply:
At present, the sea-ice handlers never use constants from the `mpassi_in` namelist (note: they would not use `mpaso_in` as you suggested above).  Presumably, the option was implemented in case they ever would need these constants as MPAS-Ocean does.